### ISSUE TITLE
Tertiary `Button` does not match its design in focused state

### ISSUE THREAD

**Issue by viktorrusakov** [openedx/paragon#2254]
On the [docs site](https://paragon-openedx.netlify.app/components/button/#core-buttons) with `edx.org` theme selected `Button` with tertiary variant has white background in focused state:
![image](https://user-images.githubusercontent.com/52399399/236144868-c69d59f9-f685-438c-8420-18122cccc60b.png)

According to [design](https://www.figma.com/file/eGmDp94FlqEr4iSqy1Uc1K/Paragon-Design-System?type=design&node-id=9076-32&t=bpFi2tGW88bny1e9-0) background should be transparent
![image](https://user-images.githubusercontent.com/52399399/236145193-6a4b328a-61a7-46a7-8e20-de75ee362c9f.png)

This is likely caused by [these styles](https://github.com/edx/brand-edx.org/blob/master/paragon/_overrides.scss#L95) in `edx.org` brand repo.

```[tasklist]
### AC
- [ ] Ensure tertiary `Button` has the same styles as in design when in focus state
- [ ] Validate that all buttons match design in all states
```

**Comment by PKulkoRaccoonGang** [comment]
PR: https://github.com/edx/brand-edx.org/pull/57

**Comment by adamstankiewicz** [comment]
@PKulkoRaccoonGang The above PR has been merged and released as v2.1.2. I assume we'll need to bump the `@edx/brand-edx.org` version for the docs site before this issue is closed?

**Comment by viktorrusakov** [comment]
Brand version was updated as a part of #2276
